plugins/tangle/consistency.go

```diff
@@ -49,6 +51,10 @@ func CheckConsistencyOfConeAndMutateDiff(tailTxHash trinary.Hash, approved map[t
 	// apply the walker diff to the cone diff
 	for addr, change := range diff {
 		coneDiff[addr] += change
+		if math.Abs(coneDiff[addr]) > int64(compressed.TotalSupply) {
+			// transaction is not consistent with the current diff because ledger changes would overflow total supply
```

Comment:
```suggestion
```

Reply:
Should I just remove the comment or use the same like in the other suggestion?
`// the mutation is not consistent with the current diff because the address would overflow/underflow from the total supply`

Reply:
Just remove the comment.